infra/dockerfile.spawn_check_pipeline

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7.3-slim
```

Comment:
Instead of defining a new image, could we make a generic "spawn" image and make this one run check-pipeline.yml and the other run data-pipeline.yml?
Would that be feasible/clean?

Reply:
Yeah that could works. The only downside I see is that the hook will contains the right CMD so we need to keep the image in sync with the hooks if we change the `/code/spawn_data_pipeline.py` command-line API.

Reply:
Filed #365 for this.